packages/app/src/lib/server/products.service.ts

```diff
@@ -632,7 +640,7 @@ export const getProductsByCatName = async (filter: ProductsFilter): Promise<Prod
 
 	const productRes = await preparedProductsByCatName.execute({
 		userId: filter.userId,
-		category: filter.category,
+		category: filter.category.toLowerCase(),
```

Comment:
This doesn't solve the problem. If you now have categories `Books` and `Books`, the ones with `Books` will never be displayed because it is still case sensitive.

Reply:
We shouldn't have a situation `Books` and `books` exist in the db at the same time because with the changes above only `books` is being written. Or do you want to catch this at the comparison in the getter?